src/decoder/decoding.worker.ts

```diff
@@ -76,16 +81,23 @@ self.onmessage = async ({ data }) => {
       }
       break;
     }
-    case EventTypes.AAC_EMITTED: {
+    case EventTypes.AAC_EMITTED:
+    case EventTypes.MPEG1AUDIO_EMITTED: {
       const { pts_timestamp, data: rawData } = data;
-    
+
       const encodedAudioChunk = new EncodedAudioChunk({
         type: 'key',
         timestamp: pts_timestamp * 1000000,
         data: rawData,
       });
 
       try {
+        if (event === EventTypes.AAC_EMITTED && currentAudioDecoderCodec !== aacAudioDecoderCodec) {
```

Comment:
Is here any reason why we move it into 
```
currentAudioDecoderCodec = codec;
```
and then we are checking `currentAudioDecoderCodec` instead of actual `codec` ?

Reply:
like
```if (event === EventTypes.AAC_EMITTED && codec !== aacAudioDecoderCodec)```